internal/infrastructure/storage/db/pg/migration/20230504102232_init.up.sql

```diff
@@ -0,0 +1,63 @@
+CREATE TABLE market (
+    name VARCHAR(50) NOT NULL PRIMARY KEY,
+    base_asset VARCHAR(255) NOT NULL,
+    quote_asset VARCHAR(255) NOT NULL,
+    base_asset_precision INTEGER NOT NULL DEFAULT 8,
+    quote_asset_precision INTEGER NOT NULL DEFAULT 8,
+    tradeable BOOLEAN NOT NULL DEFAULT FALSE,
+    strategy_type INTEGER NOT NULL DEFAULT 0,
+    base_price INTEGER NOT NULL DEFAULT 0,
+    quote_price INTEGER NOT NULL DEFAULT 0,
+    UNIQUE (base_asset, quote_asset)
+);
+
+CREATE TABLE fee (
+    id SERIAL PRIMARY KEY,
+    base_asset_fee BIGINT NOT NULL,
+    quote_asset_fee BIGINT NOT NULL,
+    type VARCHAR(10) NOT NULL CHECK (type IN ('percentage', 'fixed')),
+    fk_market_name VARCHAR(50) NOT NULL,
+    FOREIGN KEY (fk_market_name) REFERENCES market (name)
+);
+
+CREATE TABLE trade (
+    id VARCHAR(50) PRIMARY KEY,
+    type INTEGER NOT NULL,
+    fee_asset VARCHAR(255) NOT NULL,
+    fee_amount BIGINT NOT NULL,
+    trader_pubkey BYTEA NOT NULL,
```

Comment:
This is actualy not used at app level, so it should be nullable:

```suggestion
    trader_pubkey BYTEA,
```